src/app/posts/[id]/page.tsx

```diff
@@ -12,6 +13,29 @@ export async function generateStaticParams() {
   }));
 }
 
+export async function generateMetadata({ params }: Props): Promise<Metadata> {
+  const resolvedParams = await params;
+  const post = await getPostData(resolvedParams.id);
+  
+  return {
+    title: `${post.title} | Craft & Code Club`,
+    description: post.description,
+    keywords: [...post.topics, "Blog", "Artigo", "Desenvolvimento de Software", "Aprendizado", "Comunidade", "Algoritmos", "Estruturas de Dados", "System Design", "DDD"],
```

Comment:
issue: Good job! I agree with this implementation, but I believe we can improve it. For example:

![image](https://github.com/user-attachments/assets/b10337d5-15c2-4a59-b113-b490fd1e3b92)

We have de Post Tags as keywords + default Keywords. Bet for Post about Dijsktra we don't have any tag with word "Dijkstra".

My Suggestion:

![image](https://github.com/user-attachments/assets/a5194d67-15e0-430f-934b-c374e2ab6622)


On markdown for content we could have a key called "keywords", and here we use also to field the keywords

something like `keywords: [...post.keywords, ...post.topics, "Blog", "Artigo", "Desenvolvimento de Software", "Aprendizado", "Comunidade", "Algoritmos", "Estruturas de Dados", "System Design", "DDD"]`

Reply:
I completely agree with that, I think it is a good new feature.

I would suggest we merge this as it is, as a fix for the lack of specific meta tags, so we can work on another iteration for the keywords feature. Do you agree with that?

Reply:
Ok @wilsonneto-dev I will approve and we open a new PR

Reply:
Thanks. Early this week I will try to send a PR adding the keywords 🚀